t/fq-object-methods.t

```diff
@@ -2,6 +2,10 @@
 use strict;
 use Test::More;
 
+if (not $ENV{'ONLINE_TEST'}) {
```

Comment:
This particular test is a mixture of online and offline tests.  I think this should only guard the online test, but the online tests aren't co-located in the file.  If you want to shuffle the tests around and put all the tests with "fetch" in a guard, please do.  Otherwise, I'll leave this PR open and make the changes before we do the 1.50 release.